### ISSUE TITLE
JS: Error in messaging.unregisterForPushNotifications: Error: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/firebase/iid/FirebaseInstanceId;

### ISSUE THREAD

**Issue by fleminze** [nativescript-community/nativescript-push#5]
Other installed packages:

```json
  "dependencies": {
    "@angular/animations": "v13-lts",
    "@angular/common": "v13-lts",
    "@angular/compiler": "v13-lts",
    "@angular/core": "v13-lts",
    "@angular/forms": "v13-lts",
    "@angular/platform-browser": "v13-lts",
    "@angular/platform-browser-dynamic": "v13-lts",
    "@angular/router": "v13-lts",
---
---
    "@nativescript-community/insomnia": "2.0.2",
    "@nativescript-community/licenses": "2.0.7",
    "@nativescript-community/push": "^1.1.9",
    "@nativescript-community/ui-lottie": "4.3.1",
    "@nativescript-community/ui-mapbox": "6.2.15",
    "@nativescript-community/universal-links": "2.0.7",
    "@nativescript/angular": "13.0.4",
    "@nativescript/core": "8.2.4",
    "@nativescript/firebase-analytics": "file:lib/nativescript-firebase-analytics-1.1.4.tgz",
    "@nativescript/firebase-core": "file:lib/nativescript-firebase-core-1.1.4.tgz",
    "@nativescript/firebase-crashlytics": "file:lib/nativescript-firebase-crashlytics-1.1.4.tgz",
    "@nativescript/geolocation": "8.0.2",
    "@nativescript/local-notifications": "6.0.0",
    "@nativescript/theme": "3.0.2",
    "@nativescript/unit-test-runner": "3.0.4",
    "@nativescript/zip": "6.0.0",
    "@ngrx/component": "13.1.0",
    "@ngrx/effects": "13.1.0",
    "@ngrx/entity": "13.1.0",
    "@ngrx/store": "13.1.0",
    "@ngx-translate/core": "14.0.0",
    "@nota/nativescript-webview-ext": "8.0.2",
    "@nstudio/nativescript-carousel": "8.0.3",
    "aes-js": "3.1.2",
    "chalk": "4.1.2",
    "jwt-decode": "3.1.2",
    "mathjs": "10.6.1",
    "nativescript-permissions": "1.3.12",
    "pythonic": "^2.0.3",
    "ramda": "0.28.0",
    "reflect-metadata": "0.1.13",
    "rxjs": "7.5.5",
    "slate": "0.62.0",
    "spark-md5": "3.0.2",
    "url-parse": "1.5.10",
    "zone.js": "0.11.5"
  },
```

**Comment by fleminze** [comment]
Class is deprecated: https://firebase.google.com/docs/reference/android/com/google/firebase/iid/package-summary
https://github.com/nativescript-community/push/blob/3ef003357a46fb66f719eef67a759e06724fb681/plugin/platforms/android/java/com/nativescript/push/PushMessagingService.java#L79

**Comment by fleminze** [comment]
#6

**Comment by farfromrefug** [comment]
@fleminze thanks i just published 1.1.10
